src/Ilios/CoreBundle/Controller/ApplicationConfigController.php

```diff
@@ -0,0 +1,321 @@
+<?php
+
+namespace Ilios\CoreBundle\Controller;
+
+use FOS\RestBundle\Controller\Annotations\QueryParam;
+use FOS\RestBundle\Controller\Annotations\RouteResource;
+use FOS\RestBundle\Controller\Annotations as Rest;
+use FOS\RestBundle\Request\ParamFetcherInterface;
+use FOS\RestBundle\Util\Codes;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use FOS\RestBundle\Controller\FOSRestController;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use Ilios\CoreBundle\Exception\InvalidFormException;
+use Ilios\CoreBundle\Entity\ApplicationConfigInterface;
+
+/**
+ * Class ApplicationConfigController
+ * @package Ilios\CoreBundle\Controller
+ * @RouteResource("ApplicationConfigs")
+ */
+class ApplicationConfigController extends FOSRestController
+{
+    /**
+     * Get a Application configuration item.
+     *
+     * @ApiDoc(
+     *   section = "Application Configuration",
+     *   description = "Get a application configuration setting.",
+     *   resource = true,
+     *   requirements={
+     *     {
+     *        "name"="id",
+     *        "dataType"="integer",
+     *        "requirement"="\d+",
+     *        "description"="Application configuration setting identifier."
+     *     }
+     *   },
+     *   output="Ilios\CoreBundle\Entity\ApplicationConfig",
+     *   statusCodes={
+     *     200 = "A application configuration setting.",
+     *     404 = "Not Found."
+     *   }
+     * )
+     *
+     * @Rest\View(serializerEnableMaxDepthChecks=true)
+     *
+     * @param $id
+     *
+     * @return Response
+     */
+    public function getAction($id)
```

Comment:
This should use DTOs instead of entities.  We're going that way with all of the old stuff and it would be good not to have to circle back and fix new ones as well.